packages/access-mom6/package.py

```diff
@@ -21,8 +21,9 @@ class AccessMom6(CMakePackage):
     # see license file in https://github.com/ACCESS-NRI/MOM6/blob/e92c971084e185cfd3902f18072320b45d583a54/LICENSE.md
     license("LGPL-3.0-only", checked_by="minghangli-uni")
 
-    version("2025.02.001", commit="a5f4397")
-    version("2025.02.000", commit="e088c8b")
+    version("stable", branch="2025.02", preferred=True)   # need to update branch for new major versions
```

Comment:
This is ready to be updated. I'm assuming you want to do this is a subsequent PR? If not:

```suggestion
    version("stable", branch="2025.08", preferred=True)   # need to update branch for new major versions
    version("2025.08.000", tag="2025.08.000", commit="bc51c1ba407f5ae669b5bbc94b027e852e2c6ac4")
```

Reply:
Yes we can do this when making a 2025.08 release for access-om3